composer.json

```diff
@@ -20,7 +20,7 @@
         "doctrine/orm": "~2.6.0",
         "doctrine/migrations": "~1.6.0",
         "doctrine/dbal": "~2.7.0",
-        "doctrine/common": ">=2.4,<2.8-dev",
```

Comment:
The previous configuration looks "special", but I don't get the purpose of that configuration.